main.py

```diff
@@ -1 +1,15 @@
-print("Hello, world!")
+from blessed import Terminal
+
+from library import Board
+
+term = Terminal()
+board = Board()
+
+# python 3.10 feature, but it works in python 3.9
```

Comment:
Turns out I was wrong about this, I think --- more specifically, the python 3.10 feature that works in 3.9 is splitting multiple context managers over multiple lines, so this comment is wrong.

Reply:
Removed comment

Reply:
Looks like you forgot to commit this?